app.py

```
@@ -0,0 +1,136 @@
+
+@app.get("/")
+def root():
+    return {"message": "Server is running"}
+
+
+# 문서 노드
+@app.post("/node/{projectId}/document")
+def document_node(projectId: str, req: DocumentNodeRequest):
+    try:
+        logging.info(f"[문서 노드] PDF 다운로드 시작: {req.object_key}")
+        response = requests.get(req.object_key)
```

Ruff (0.13.1)

Probable use of `requests` call without timeout

(S113)